packages/test/src/components/swap/TokenSwapBak.tsx

```
@@ -0,0 +1,305 @@
+        btcAddress,
+        strategyId,
+      },
+    });
+
+    if (res?.error) throw new Error(res.error);
+
+    return res;
+  };
+
+  // Main swap function to handle swap click
+  const handleSwap = async () => {
```

ESLint

[error] 'handleSwap' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)